pkgs/development/python-modules/patsy/default.nix

```diff
@@ -0,0 +1,34 @@
+{ lib
+, fetchurl
+, buildPythonPackage
+, nose
+, six
+, numpy
+, scipy
+}:
+
+buildPythonPackage rec {
+  pname = "patsy";
+  version = "0.4.1";
+
+  src = fetchurl {
```

Comment:
why not from PyPI?

Reply:
I tried it actually but specifically patsy `0.4.1` is not found there